### DISCUSSION TITLE
0.8.4: Fix Transmission Bug Lihuiyu Cards

### DISCUSSION THREAD

**Discussion by tatarize** [meerk40t/meerk40t#1858] [Announcements]
## What's Changed
* Parse .ezd files. by @tatarize in https://github.com/meerk40t/meerk40t/pull/1763
* Add an undo mark between elements loaded and operations replaced. by @tatarize in https://github.com/meerk40t/meerk40t/pull/1801
* Fix invalid tick_distance use by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1803
* Fix absolute wordlist reference offset by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1804
* Divide Internal/External plugins into files. Prep for Builds. by @tatarize in https://github.com/meerk40t/meerk40t/pull/1805
* Placements by @tatarize in https://github.com/meerk40t/meerk40t/pull/1785
* Reestablishes unknown operation type treatment by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1806
* Add validate function to 'place point' by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1807
* Fix wordlist crash with pure numbers by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1808
* Add right mouse action to wordlist button by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1809
* Fix non used-tooltip for multi-buttons by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1810
* Language Updates by @tatarize in https://github.com/meerk40t/meerk40t/pull/1815
* Fix plan update crash in Laserpanel by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1817
* Translation Errors, language flag and printf commands by @tatarize in https://github.com/meerk40t/meerk40t/pull/1819
* Use existing pane for size in transformation by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1824
* Change `maintain_zoom_reseize` to `autofocus_resize` default off by @tatarize in https://github.com/meerk40t/meerk40t/pull/1823
* Project Focus: Display by @tatarize in https://github.com/meerk40t/meerk40t/pull/1811
* Some image fixes by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1825
* Some changes around reference object by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1826
* Fix image translation in keyhole by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1827
* Add context menu option for grid placements by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1828
* Fix magnet code by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1830
* Magnets + Fixes for Newly by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1833
* Magnets part 2 by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1834
* Make magnets persistent by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1832
* Fix icon+label display after operation load by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1836
* Visual feedback during planning stages by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1835
* Fix tracegen command by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1838
* Simplify task for Welzl by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1840
* Ch341 Checks by @tatarize in https://github.com/meerk40t/meerk40t/pull/1820
* Deprecate const state values, use strings by @tatarize in https://github.com/meerk40t/meerk40t/pull/1843
* Newly - Update 2 by @tatarize in https://github.com/meerk40t/meerk40t/pull/1814
* Fix Issue #1829,  do not yield empty cutgroups by @tatarize in https://github.com/meerk40t/meerk40t/pull/1845
* Smaller cosmetical adjustments by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1846
* CH341 M3Nano Status by @tatarize in https://github.com/meerk40t/meerk40t/pull/1844
* Inkscape unicode issue, translations by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1848
* Position the Machine Origin Widget at the device origin (always). by @tatarize in https://github.com/meerk40t/meerk40t/pull/1849
* Grbl Write Lock by @tatarize in https://github.com/meerk40t/meerk40t/pull/1625
* Translation typo by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1851
* Correct PySerial Requirement Fix - Implement TCP by @tatarize in https://github.com/meerk40t/meerk40t/pull/1850
* Add omitted translation by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1853
* Fix 'simplify' by bypassing faulty code for now by @jpirnay in https://github.com/meerk40t/meerk40t/pull/1852
* Correct GRBL sync locks for Controller by @tatarize in https://github.com/meerk40t/meerk40t/pull/1854
* Correct GRBL Pause/Resume and read buffered values. by @tatarize in https://github.com/meerk40t/meerk40t/pull/1855
* Test GRBL with two independent threads for reading and writing. by @tatarize in https://github.com/meerk40t/meerk40t/pull/1856
* Fix the TCP output for GRBL devices by @tatarize in https://github.com/meerk40t/meerk40t/pull/1857

**Full Changelog**: https://github.com/meerk40t/meerk40t/compare/0.8.3000...0.8.4000

<hr /><em>This discussion was created from the release <a href='https://github.com/meerk40t/meerk40t/releases/tag/0.8.4000'>0.8.4: Fix Transmission Bug Lihuiyu Cards</a>.</em>

**Comment by FaithCustom** [comment]
Mac OS X version when will it be available?

**Reply by tatarize** [reply]
Probably when @joerlane gets around to executing the runners. In retrospect the beta would have given you something to run and had mac version. So I now regret removing it as I do for most betas when the full versions release.

**Reply by FaithCustom** [reply]
Thank you very much Tatarize, for making the Mac version available, the interface is getting better and better.
